SECURITY.md

```diff
@@ -0,0 +1,98 @@
+- The Fix Lead will cherry-pick the patches onto the main branch and all relevant release branches. The Fix Team will LGTM and merge. Maintainers will merge these PRs as quickly as possible. Changes shouldn't be made to the commits, even for a typo in the CHANGELOG, as this will change the git SHA of the commits leading to confusion and potentially conflicts as the fix is cherry-picked around branches.
+- The Fix Lead will request a CVE from [DWF](https://github.com/distributedweaknessfiling/DWF-Documentation) and include the CVSS and release details.
+- The Fix Lead will email kyverno@googlegroups.com now that everything is public announcing the new releases, the CVE number, and the relevant merged PRs to get wide distribution and user action. As much as possible this email should be actionable and include links on how to apply the fix to user's environments; this can include links to external distributor documentation.
+- The Fix Lead will remove the Fix Team from the private security repo.
+
+### Retrospective
+These steps should be completed 1-3 days after the Release Date. The retrospective process [should be blameless](https://landing.google.com/sre/book/chapters/postmortem-culture.html).
+- The Fix Lead will send a retrospective of the process to kyverno@googlegroups.com, including details on everyone involved, the timeline of the process, links to relevant PRs that introduced the issue, if relevant, and any critiques of the response and release process.
+- Maintainers and Fix Team are also encouraged to send their own feedback on the process to kyverno@googlegroups.com. Honest critique is the only way we are going to get good at this as a community.
+
+
+### Embargo Policy
+The information members receive on kyverno@googlegroups.com must not be made public, shared, nor even hinted at anywhere beyond the need-to-know within your specific team except with the list's explicit approval. This holds true until the public disclosure date/time that was agreed upon by the list. Members of the list and others may not use the information for anything other than getting the issue fixed.
+
+Before any information from the list is shared with respective members of your team required to fix the said issue, they must agree to the same terms and only find out information on a need-to-know basis.
+
+In the unfortunate event you share the information beyond what is allowed by this policy, you _must_ urgently inform the kyverno-security@googlegroups.com mailing list of precisely what information leaked and to whom. A retrospective will take place after the leak so we can assess how to not make the same mistake in the future.
+
+If you continue to leak information and break the policy outlined here, you will be removed from the list.
+
+
+### Contributing Back
```

Comment:
Not exactly sure what this means....can you please elaborate on the intent?

Reply:
This is in general with respect to the security of the project. If anybody wants to contribute in the security aspect of Kyverno, they can help to do so in technical as well as administrative ways.